### PR TITLE
Enable integration tests again

### PR DESCRIPTION
**Pull request by peterfication**
This PR enables two integration tests again. If this is merged in, I will work on the other commented out integration tests.

### CODE REVIEW

File: lua/textcase/init.lua [hunk 1]
```diff
@@ -35,6 +35,7 @@ local M = {
     toggle_boolean = presets.toggle_boolean,
   },
   setup = presets.setup,
+  init = presets.Initialize,
```
Comment (peterfication) [inline lua/textcase/init.lua:38]: I exposed it for the integration tests to work and then I adjusted it in start.vim as well. Does this make sense?

File: plugin/start.vim [hunk 2]
```diff
@@ -7,15 +7,15 @@ command! -range TextCaseOpenTelescopeLSPChange <line1>,<line2>lua require("textc
 command! -range TextCaseStartReplacingCommand <line1>,<line2>lua require("textcase").start_replacing_command()
 
 
-function! TextCaseSubstituteLauncher(...) range 
+function! TextCaseSubstituteLauncher(...) range
```
Comment (peterfication) [inline plugin/start.vim:10]: This diff is just removing spaces at the end of the lines.

**Comment by peterfication** [comment]
@johmsalas please let me know if this change makes sense. It still takes me some effort to understand how Neovim plugins exactly work and how they are loaded.

File: lua/textcase/init.lua (hunk 1 above)
Comment (johmsalas) [reply lua/textcase/init.lua:38]: Should be fine

**Comment by johmsalas** [comment]
I thought the start.vim was loaded by the test runner by default. As long as it works I'm approving, we can get more technical later :)

**Review by johmsalas — APPROVED** [review]
LGreatTM

**Pull request merged**